private/buf/cmd/buf/command/push/push.go

```diff
@@ -185,7 +186,7 @@ func run(
 	case 0:
 		return nil
 	case 1:
-		_, err := container.Stdout().Write([]byte(uuidutil.ToDashless(commits[0].ModuleKey().CommitID()) + "\n"))
+		_, err := container.Stdout().Write([]byte(commits[0].ModuleKey().CommitID().String() + "\n"))
```

Comment:
I think this will print the commit in dashful format, and what we want is print them all in dashless.